Library/Formula/mesos.rb

```diff
@@ -65,5 +139,11 @@ def install
     Process.kill("TERM", master)
     Process.kill("TERM", slave)
     assert File.exist?("#{testpath}/executed")
+
+    user_site = Language::Python.user_site_packages("python")
+    mkdir_p user_site
+    pth_contents = "import site; site.addsitedir('#{Language::Python.homebrew_site_packages}')\n"
+    (user_site/"homebrew.pth").atomic_write pth_contents
+    system "python", "-c", "import mesos.native"
```

Comment:
I agree. It looks like this touches a user's files, but it doesn't; $HOME is set to a temporary path for tests and Python respects it.